ui-v2/app/controllers/dc/services/index.js

```diff
@@ -37,17 +37,17 @@ export default Controller.extend(WithHealthFiltering, {
       item.hasStatus(status)
     );
   },
-  totalWidth: computed('{maxPassing,maxWarning,maxCritical}', function() {
+  maxWidth: computed('{maxPassing,maxWarning,maxCritical}', function() {
     const PADDING = 32 * 3 + 13;
     return ['maxPassing', 'maxWarning', 'maxCritical'].reduce((prev, item) => {
       return prev + width(get(this, item));
     }, PADDING);
   }),
-  thWidth: computed('totalWidth', function() {
-    return widthDeclaration(get(this, 'totalWidth'));
+  totalWidth: computed('maxWidth', function() {
+    return widthDeclaration(get(this, 'maxWidth'));
   }),
```

Comment:
The name `thWidth` no longer makes sense (I was using it just to set the width of the `<th>` but now I'm also using it to set a `<td>`, changed the name to `totalWidth` as thats what it is (move the old `totalWidth` to `maxWidth`)